src/Components/Aspire.Confluent.Kafka/MetricsService.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.Text.Json;
+using Microsoft.Extensions.Hosting;
+using static Aspire.Confluent.Kafka.ConfluentKafkaMetrics;
+
+namespace Aspire.Confluent.Kafka;
+
+internal sealed class MetricsService(MetricsChannel channel, ConfluentKafkaMetrics metrics) : BackgroundService
```

Comment:
Can we add some basic summary comments about what these channel, service, etc classes are/do? So someone can get a quick summary of what they represent.

Reply:
done